windows/translate/nl.nsi

```diff
@@ -0,0 +1,37 @@
+; Dutch translation
+
+LangString DESKTOP_SET_SHORTCUT ${LANG_DUTCH} \
+    "Desktop shortcut maken"
+
+LangString INVALID_FOLDER_MSG ${LANG_DUTCH} \
+    "De doelmap bestaat al. Kies een nieuwe doelmap."
+
+LangString RUNNING_APP_MSG ${LANG_DUTCH} \
+    "${APP_NAME} loopt nog. Sluit het programma en start de installatie opnieuw.."
```

Comment:
Are two dots correct here?

Reply:
Nice catch @ann0see. Should be single dot.

Reply:
Great! Thanks